main.go

```diff
@@ -553,3 +552,26 @@ func abs(i int) int {
 	}
 	return i
 }
+
+func createClockEmoji(t time.Time, loc *time.Location) string {
```

Comment:
Simply take `t` here.

Also, you might want to take `*time.Time` to avoid cloning a time object.
But yeah, you decide.